translations/strings.yaml

```diff
@@ -69,7 +74,7 @@ settingsDev:
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
 
-splittunnel:
+splitTunnel:
```

Comment:
This is going to break existing code? Strings are not exposed yet, so it's not an issue for l10n.

Reply:
This is fixed by my commit.